spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/web/ServerProperties.java

```diff
@@ -1181,6 +1218,113 @@ else if (handler instanceof HandlerCollection) {
 			});
 		}
 
+		public static class Accesslog {
+
+			/**
+			 * Enable access log.
+			 */
+			private Boolean enabled;
+
+			/**
+			 * Jetty accesslog filename.
+			 */
+			private String filename;
+
+			/**
+			 * the number of days before rotated log files are deleted.
```

Comment:
Default to 31. Documentation for keys do not start with "the", "a", etc. Check other keys for inspiration.